apps/web/public/static/locales/en/common.json

```diff
@@ -1958,6 +1958,7 @@
   "members_default_schedule_description": "We will use each members default availability schedule. They will be able to edit or change it.",
   "requires_at_least_one_schedule": "You are required to have at least one schedule",
   "default_conferencing_bulk_description": "Update the locations for the selected event types",
+  "default_schedules_bulk_description": "Update the schedules for the selected event types",
```

Comment:
Add this key in for other locales as well

Reply:
So do I have to add it manually or is there any autmoated way? @anikdhabal @Amit91848

Reply:
I used this script: translate-locales
and a lot of translation were converted aprt form my own key.

Reply:
Hey @SySagar, you don't need to add it to other locales. It is auto-added; currently, due to some issue, it is not. But we will fix it shortly

Reply:
I wanted to know what is this use of "translate-locales"? is it buggy?